sql/core/src/main/scala/org/apache/spark/sql/execution/Columnar.scala

```diff
@@ -439,7 +438,7 @@ case class RowToColumnarExec(child: SparkPlan) extends RowToColumnarTransition {
   )
 
   override def doExecuteColumnar(): RDD[ColumnarBatch] = {
-    val enableOffHeapColumnVector = sqlContext.conf.offHeapColumnVectorEnabled
```

Comment:
The previous code is better, as `sqlContext` is kind of "materialized", which is the one that was used to create the plan node. `conf` always gets the active one.

Reply:
OK. will revert.